### PR TITLE
LPS-123608 Convert discard view to use modal rather than react render for viewing entries

### PR DESCRIPTION
**Pull request by stsquared99**
@Preston-Crary Talked it over a little with PJ, decided the clearer design is to use a modal for this view as well.

### CODE REVIEW

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewDiscardDisplayContext.java [hunk 1]
```diff
@@ -88,6 +91,20 @@ public ViewDiscardDisplayContext(
...
 				for (CTEntry ctEntry : ctEntries) {
+					RenderURL viewURL = _renderResponse.createRenderURL();
+
+					viewURL.setParameter(
+						"mvcRenderCommandName", "/change_tracking/view_diff");
+					viewURL.setParameter(
+						"ctEntryId", String.valueOf(ctEntry.getCtEntryId()));
+
+					try {
+						viewURL.setWindowState(LiferayWindowState.POP_UP);
```
Comment (Preston-Crary) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewDiscardDisplayContext.java:102]: There's a few of these now, let's move this setWindowState exception suppression block in PublicationsPortletURLUtil.

**Comment by Preston-Crary** [comment]
Just push that one little change and I'll forward it.

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewDiscardDisplayContext.java (hunk 1 above)
Comment (Preston-Crary) [reply modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewDiscardDisplayContext.java:102]: `void setWindowState(PortletURL, WindowState)`

**Comment by stsquared99** [comment]
@Preston-Crary Updated, thanks

**Comment by Preston-Crary** [comment]
ci:forward

**Pull request closed without merging**